clients/src/main/resources/common/message/LeaveGroupResponse.json

```diff
@@ -19,11 +19,21 @@
   "name": "LeaveGroupResponse",
   // Version 1 adds the throttle time.
   // Starting in version 2, on quota violation, brokers send out responses before throttling.
-  "validVersions": "0-2",
+  // Starting in version 3, we will make leave group request into batch mode.
+  "validVersions": "0-3",
   "fields": [
     { "name": "ThrottleTimeMs", "type": "int32", "versions": "1+", "ignorable": true,
       "about": "The duration in milliseconds for which the request was throttled due to a quota violation, or zero if the request did not violate any quota." },
-    { "name": "ErrorCode", "type": "int16", "versions": "0+",
-      "about": "The error code, or 0 if there was no error." }
+    { "name": "ErrorCode", "type": "int16", "versions": "0-2",
```

Comment:
Do we need to make this ignorable or add a default? I think it would be helpful to add some tests to `MessageTest` to make version expectations clear.